README.md

```diff
@@ -1,13 +1,85 @@
-### Installation
+![img.png](img.png)
+
+Obsidian Gateway - a fast and reliable portal to Arweave SmartWeave interaction transactions.
+- fast - load your contract interactions in seconds, not minutes!
```

Comment:
Out of curiosity, what's our response time?

Reply:
for our loot contract and current configuration (5000 interactions per page)
https://d1o5nlqr4okus2.cloudfront.net/gateway/interactions?contractId=Daj-MNSnH55TDfxqC7v4eq0lKzVIwh98srUaWqyuZtY&confirmationStatus=confirmed&page=1
https://d1o5nlqr4okus2.cloudfront.net/gateway/interactions?contractId=Daj-MNSnH55TDfxqC7v4eq0lKzVIwh98srUaWqyuZtY&confirmationStatus=confirmed&page=2

- if you hit the CDN cache (which has 130s TTL) it's usually less than 100ms in total (usually around 50-60ms)
- if you don't hit the the CDN cache - it's usually around 500-600ms, up to ~1s.

There's probably still some room for improvement in the sql query and probably indexes configuration.